sympy/codegen/tests/test_rewriting.py

```diff
@@ -169,3 +169,5 @@ def test_create_expand_pow_optimization():
 
     sin4x = sin(x)**4
     assert ccode(optimize(sin4x, [my_opt])) == 'pow(sin(x), 4)'
+
+    assert ccode(optimize((x**(-4)), [my_opt])) == 'pow(x, -4)'
```

Comment:
OK, that looks better. Eventually we should probably see if we can make it output `1.0/(x*x*x*x)` for this case. But at least this form is not incorrect.